libs/ach-positive-pay/.eslintrc.json

```diff
@@ -1,20 +1,13 @@
 {
```

Comment:
Is this file a copy of transfer-journey, .eslintrc.json? It seems it has been renamed. Can you confirm both does exist?

Reply:
I'm not sure I understand your question, but all of the linting is working correctly.